rest-api/api/awardee_api.py

```diff
@@ -0,0 +1,24 @@
+from api.base_api import BaseApi
+from app_util import auth_required
+from api_util import PTC_AND_HEALTHPRO
+from dao.hpo_dao import HPODao
+from werkzeug.exceptions import NotFound
+
+class AwardeeApi(BaseApi):
+  def __init__(self):
+    super(AwardeeApi, self).__init__(HPODao(), get_returns_children=True)
+
+  @auth_required(PTC_AND_HEALTHPRO)
+  def get(self, a_id=None):
+    if a_id:
+      hpo = self.dao.get_by_name(a_id)
+      if not hpo:
+        raise NotFound("Awardee with ID %s not found" % a_id)
+      return self._make_response(self.dao.get_with_children(hpo.hpoId))
+    else:
```

Comment:
opt: my Go style training tells me to drop the unnecessary `else` (or flip the conditional and unindent the large codeblock)

Reply:
Done.